java/src/jmri/jmrix/AbstractThrottleManager.java

```diff
@@ -830,4 +824,4 @@ boolean containsListener(ThrottleListener l) {
     }
 
     private final static Logger log = LoggerFactory.getLogger(AbstractThrottleManager.class);
-}
+}
```

Comment:
In addition to the things @rhwood mentioned, there is some strange character at the end of the file ( I can’t tell what it is from the web interface)

Reply:
Lack of CRLF before EOF.  Added CRLF in subsequent commit.